data/libexec/host-runner

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+#
+# Copyright © 2019 – 2020 Red Hat Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+executable=$(basename $0)
+priviledged=false
+
+if [ $# -ge 1 ]; then
+	case "$1" in
+		--host-runner-sudo )
+			shift
+			priviledged=true
+	esac
+fi
+
+if $priviledged; then
+	executable="sudo $executable"
+fi
+
+exec flatpak-spawn --host --watch-bus --forward-fd=1 --forward-fd=2 --directory=$(pwd) --env=TERM=xterm-256color $executable "$@"
```

Comment:
I am curious about the use of `--watch-bus`.  It's not wrong, but given that `flatpak-spawn(1)` always forwards signals, I am curious if you came across a scenario where the container-side shim would cleanly exit without signals leaving the host-side running.  I am trying to understand if we should use `--watch-bus` more widely.